src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/BalanceSheetController.java

```diff
@@ -1,4 +1,49 @@
 package uk.gov.companieshouse.web.accounts.controller.smallfull;
 
+import javax.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import uk.gov.companieshouse.web.accounts.model.smallfull.BalanceSheet;
+import uk.gov.companieshouse.web.accounts.service.smallfull.BalanceSheetService;
+
+@Controller
 public class BalanceSheetController {
-}
+
+    private static final String BALANCE_SHEET_PATH = "/company/{companyNumber}/transaction/{transactionId}/company-accounts/{companyAccountsId}/small-full/balance-sheet";
+
+    private static final String SMALL_FULL_BALANCE_SHEET = "smallfull/balanceSheet";
+
+    @Autowired
+    private BalanceSheetService balanceSheetService;
+
+    @RequestMapping(value = BALANCE_SHEET_PATH, method = RequestMethod.GET)
```

Comment:
This can be changed from `@RequestMapping` to use the Get one instead `@GetMapping` which means you won't need the `RequestMethod.GET` anymore :)